app/scripts/lib/bloomfilter.js

```diff
@@ -0,0 +1,118 @@
+define([
+],
+function() {
+  'use strict';
+  var typedArrays = typeof ArrayBuffer !== "undefined";
```

Comment:
@vladikoff @pdehaan and @rfk https://github.com/TDA/fxa-bloom-filter. I created a separate repo as a bower thingy. So fxa-bloom-filter or fxa-password-strength-checker ?